jest.config.js

```diff
@@ -2,7 +2,8 @@ const { jestConfig } = require('@salesforce/sfdx-lwc-jest/config');
 module.exports = {
     ...jestConfig,
     moduleNameMapper: {
-        '^(c)/(.+)$': '<rootDir>/src/lwc/$2/$2'
+        '^(c)/(.+)$': '<rootDir>/src/lwc/$2/$2',
+        "^lightning/navigation$": "<rootDir>/tests/__mocks__/lightning/navigation"
```

Comment:
A path has been added for the Jest navigation testing.
@dcaballeroSf , @bdvorachek  , @psadie-sfdo , @kenlewis-sfdo